src/hotspot/share/cds/dynamicArchive.cpp

```diff
@@ -170,7 +170,6 @@ void DynamicArchiveBuilder::init_header() {
   assert(FileMapInfo::dynamic_info() == mapinfo, "must be");
   _header = mapinfo->dynamic_header();
 
-  Thread* THREAD = Thread::current();
```

Comment:
Unused?  Looks like you could also remove these in a trivial pre-commit.

Reply:
Yes unused. Again trying to avoid further delays. Also I think this code just went through an undo and will soon be redone, in which case if it gets integrated first I'll try to get this cleaned up. Thanks.